appium/webdriver/webdriver.py

```diff
@@ -776,16 +773,15 @@ def set_value(self, element, value):
         return self
 
     def pull_file(self, path):
-        """Retrieves the file at `path`. Returns the file's content encoded as
-        Base64.
+        """Retrieves the file at `path`. Returns the file's contents.
 
         :Args:
          - path - the path to the file on the device
         """
         data = {
             'path': path,
         }
-        return self.execute(Command.PULL_FILE, data)['value']
+        return base64.b64decode(self.execute(Command.PULL_FILE, data)['value'])
```

Comment:
I would say we should not do this, since it is going to break the backward compatibility.

Reply:
You're right I overlooked this, it's removed